Doc/whatsnew/3.14.rst

```diff
@@ -2797,6 +2797,8 @@ Deprecated
 
 .. include:: ../deprecations/c-api-pending-removal-in-3.15.rst
 
+.. include:: ../deprecations/c-api-pending-removal-in-3.16.rst
```

Comment:
We'll need to do the same for 3.15 but I don't think we already have the sections ready (maybe we do)